### ISSUE TITLE
How to set cV, ingest PartA extension, user#authId through SDK?

### ISSUE THREAD

**Issue by TedaLIEz** [microsoft/cpp_client_telemetry#1181]
Hi team, 
I'm trying to figure out the difference of our telemetry between via SDK before and after, I found some of the properties are missing after I migrate to 1DS SDK:

1. Record#cV
2. Record#ingest
3. Record#user#authId

I'm wondering if the Android client can set those properties through API? Those properties are already defined in the bond schema, but I haven't figured out how to assign those properties.

**Comment by TedaLIEz** [comment]
@lalitb friendly ping for this issue, could you please help?

**Comment by anod** [comment]
We don't use OneCollector, I'm not familiar with its properties

**Comment by TedaLIEz** [comment]
@anod @lalitb I was wondering if you're familiar with the Common Schema. We're trying to align the data before and after using the 1DS SDK to minimize the impact.

Currently, we're using the old CLL library as our telemetry framework, which sends data to the Vortex in CS 2.1 format. We're planning to emit events via 1DS to the Aria in CS 4.0 format.

If you have any knowledge or resources related to this, please let me know. Thank you!

**Comment by lalitb** [comment]
@TedaLIEz - 

If you are trying to set the authId for the User part A extension - it should be accessible via custom decorator concept. Search API examples for "Decorator".

To set thecv value, you can use `event_name.SetProperty(CorrelationVector::PropertyName, corr_vec_value)`;

**Comment by TedaLIEz** [comment]
@lalitb Yes, I've noticed this `CorrelationVector::PropertyName` and I tried to set this property with `SetProperty` in Kotlin wrapper API, but I failed to get the value in the Aria via OneCollector endpoint. I'll try to reach out to the Aria support team for confirmation.


Another topic here is I don't see any code related to the `extIngest` in the record structure, do you have the plan to support this extension?

**Comment by TedaLIEz** [comment]
And I don't find any API related to set authId in the EventProperties.
